main-app/tests/audio-tests/audioSpriteTests.js

```diff
@@ -8,12 +8,29 @@
          audioSprite;
 
         beforeEach(function() {
-            module('tombola.noughtsAndCrosses.audioSprite');
+            module('tombola.noughtsAndCrosses.audio');
 
-            inject(function (_$injector_) {
-                $timeout = _$injector_;
+            inject(function ($injector) {
+                $timeout = $injector.get('$timeout');
+                $document = $injector.get('$document');
+                audioSprite = $injector.get('audioSprite');
             });
         });
 
+        it("Ensure audioSprite it set correctly", function() {
+            var createSprite = function() {
+
+                var doc = $document[0];
```

Comment:
I'd collapse this down to:
    var htmlBodyChildren= $document[0].body.children;
You don't need body or document more than once, so they are needless. They are useful for debugging when you are writing the code, you can check whether you have the right bit of the document, afterwards not so much.